### PR TITLE
Automatic commit: Move 'cloudsdk-virtual-data-model-odata' from QA to…

### PR DESCRIPTION
**Pull request by nabilshafi1**
Modified the tutorial

### CODE REVIEW

**Comment by nabilshafi1** [comment]
Could you please merge it

**Comment by sacnl** [comment]
Hi @thecodester , is it ok if I merge this PR?

**Comment by thecodester** [comment]
@sacnl Will review tomorrow and handle then. I wrote email to Nabil and Karsten regarding getting information about the missions you planned.

File: tutorials/cloudsdk-ci-cd/cloudsdk-ci-cd.md [hunk 1]
```diff
@@ -0,0 +1,252 @@
+---
+author_name: Benjamin Heilbrunn
```
Comment (thecodester) [inline tutorials/cloudsdk-ci-cd/cloudsdk-ci-cd.md:2]: There is no author URL here -- we require both name and URL

File: tutorials/cloudsdk-ci-cd/cloudsdk-ci-cd.md [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+title: Cloud SDK Continuous Integration and Delivery
+description: Learn how to use the SAP Cloud SDK Continuous Integration and Continuous Delivery toolkit to build, test, and deploy your project.
+time: 30
+tags: [tutorial>intermediate, products>sap-s-4hana-cloud-sdk]
+primary_tag: products>sap-s-4hana-cloud-sdk
+---
+
+## Prerequisites
+For getting started with Continuous Integration and Delivery (`Cx`) in your project, you need to assure the following prerequisites.
```
Comment (thecodester) [inline tutorials/cloudsdk-ci-cd/cloudsdk-ci-cd.md:11]: Put this as bullet

File: tutorials/cloudsdk-ci-cd/cloudsdk-ci-cd.md [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+**Linux host with Docker**: For instantiating the SAP Cloud SDK Cx-Server, you need to provide a suitable host or virtual machine with a Linux operating system and Docker installed. Please also ensure that the user with whom you start the Cx-Server belongs to the docker group.
```
Comment (thecodester) [inline tutorials/cloudsdk-ci-cd/cloudsdk-ci-cd.md:12]: Put this (and next item) in bullet and indent 4 spaces (subbullet)

File: tutorials/cloudsdk-ci-cd/cloudsdk-ci-cd.md [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+**Project sources in GitHub or Git**: Your project source files need to be available on a git or `Github server`, which is accessible from the Cx-Server host. Creating your project is explained in [Step 2](https://blogs.sap.com/2017/05/21/step-2-with-sap-s4hana-cloud-sdk-helloworld-on-scp-classic/) and [Step 3](https://blogs.sap.com/2017/05/19/step-3-with-sap-s4hana-cloud-sdk-helloworld-on-scp-cloudfoundry/).
+
+>The SAP Cloud SDK Pipeline uses Docker images for each individual build step. Accordingly, you do not need to take care of installing any further dependencies on your host machine. All tools required for building, testing, quality checking, and deploying your applications, are dynamically retrieved in the form of Docker images.
+
+## Details
+### You will learn
+  - How to develop and release your application in short cycles.
```
Comment (thecodester) [inline tutorials/cloudsdk-ci-cd/cloudsdk-ci-cd.md:19]: No periods at end of How to ...

File: tutorials/cloudsdk-ci-cd/cloudsdk-ci-cd.md [hunk 5]
```diff
@@ -0,0 +1,252 @@
...
+  - How to install and run cx-server.
+  - How to setup a Jenkins job.
+
+---
+
+[ACCORDION-BEGIN [Step 1: ](Installing and Managing the Cx-Server)]
```
Comment (thecodester) [inline tutorials/cloudsdk-ci-cd/cloudsdk-ci-cd.md:25]: All steps are simple verbs and sentence case, so make this Install and manage ... 

Do this for other steps.

Step 1 is quite long -- can we divide into several steps?

File: tutorials/cloudsdk-ci-cd/cloudsdk-ci-cd.md [hunk 6]
```diff
@@ -0,0 +1,252 @@
...
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 2: ](Building your first project)]
+After configuring and starting your Cx-Server instance, you are ready to build, test, and deploy your first cloud application. This application needs to be created from one of the SAP Cloud SDK archetypes.
+
+Before creating your build job, you need to configure your SCM system of choice. In our example, we will assume that your project is located in GitHub Enterprise. Alternatively, you can also skip the next step (Setup GitHub Enterprise) and use your account on github.com.
+
+>Jenkins also supports other SCM systems such as GitLab or plain git. You are free to install the corresponding plugin to enable your custom SCM setup.
+
+### Setup GitHub Enterprise
```
Comment (thecodester) [inline tutorials/cloudsdk-ci-cd/cloudsdk-ci-cd.md:153]: "Set up" is 2 words

File: tutorials/cloudsdk-ci-cd/cloudsdk-ci-cd.md [hunk 7]
```diff
@@ -0,0 +1,252 @@
...
+
+On this screen click on `Add Credentials` and enter username, password, as well as a ID for your credentials record.
+
+![Add Credentials](jenkis-add-credentials.png)
+
+As a result, navigating to `Credentials` should show your freshly created entry:
+
+![Jenkins Credentials](jenkins-credentials.png)
+
+Finally, you can now leverage this credentials record by adding the credentials configuration property to the `integrationTests` stage of your `pipeline_config.yml`. The example below shows am example of mapping it to the system alias `MyErpSystem`, which you defined earlier in your `systems.yml` or `json` file.
```
Comment (thecodester) [inline tutorials/cloudsdk-ci-cd/cloudsdk-ci-cd.md:187]: example is written twice and there is typo ("am") -- if possible, get tutorial copyedited.

**Pull request closed without merging**